### ISSUE TITLE
Maybe Encode problematic text using ROT13

### ISSUE THREAD

**Issue by michaelpacer** [amperser/proselint#526]
Possibly look to this.py or just implement it to avoid people's firewalls and dirty word blockers.

But it'd make it harder for new people to code. I'm conflicted.

**Comment by suchow** [comment]
What problematic text are you referring to?

**Comment by Decagon** [comment]
Do you mean encoding the swear words in ROT13, so that someone couldn't get around the filter by using ROT13?

**Comment by suchow** [comment]
My guess is that some developers' firewalls and dirty-word blockers are preventing them from seeing the source code to modules such as [`cursing.nfl`](https://github.com/amperser/proselint/blob/master/proselint/checks/cursing/nfl.py). @michaelpacer's suggestion is a way to maneuver around these blockers.

**Comment by michaelpacer** [comment]
Yes that and things like [lgbtq.offensive_terms](https://github.com/amperser/proselint/blob/master/proselint/checks/lgbtq/offensive_terms.py) 

Also my concern was in part based on previous complaints that we were being banned from high schools

My hesitation stems from the fact that it would make these modules specifically more difficult to contribute to, which has the adverse effect of slowing the progress on those modules…
